examples/applications/restapi/handler_utils.py

```diff
@@ -444,7 +446,7 @@ async def _create_tx_helper(self, request) -> Union[Tuple, Fault]:
     async def _broadcast_transaction(self, rawtx: str, tx_hash: bytes, account: AbstractAccount):
         result = await self.send_request('blockchain.transaction.broadcast', [rawtx])
         account.set_transaction_state(tx_hash=tx_hash,
-                                      flags=TxFlags.StateDispatched | TxFlags.HasByteData)
+                                      flags=TxFlags(TxFlags.StateDispatched | TxFlags.HasByteData))
```

Comment:
TxFlags is now an IntFlag, not an IntEnum, this means that it retains it's type for combinations.
```
>> TxFlags.StateDispatched | TxFlags.StateReceived
TxFlags(StateDispatched|StateReceived)
```